src/main/java/uk/gov/hmcts/cmc/claimstore/services/ccd/legaladvisor/DocAssemblyTemplateBodyMapper.java

```diff
@@ -163,6 +168,31 @@ public DocAssemblyTemplateBody changeContactBody(CCDCase ccdCase) {
             .build();
     }
 
+    public DocAssemblyTemplateBody paperDefenceForm(CCDCase ccdCase) {
+        CCDRespondent defendant = ccdCase.getRespondents().get(0).getValue();
+        String defendantName = defendant.getClaimantProvidedPartyName();
+        CCDAddress defendantAddress = getDefendantAddress(defendant);
+
+        CCDApplicant claimant = ccdCase.getApplicants().get(0).getValue();
+
+        LocalDate extendedResponseDeadline =
```

Comment:
I think response deadline is reset here, not extended, so it be updated in ccdcase and picked from there

Reply:
Will double check the requirements but it is my understanding that we need both the original deadline and the extended. But you are correct in pointing out that this date may have already been extended.

Reply:
Actually as the form will get generated when the claimant submits the claim then the defendant will not have had a chance to extend the deadline.

Reply:
Form has option to extend deadline, so they can return part of the form with new deadline request